### PR TITLE
add sign out functionality / create dynamic menu

### PR DESCRIPTION
**Pull request by ivailo-devrix**

### CODE REVIEW

File: includes/theme-compat/header.php
Comment (metodiew) [inline includes/theme-compat/header.php]: if this is a nested `if` we have the tab so it can be visible that's art of the if statement

File: index.php
Comment (metodiew) [inline index.php]: we have mixed coding styles, are we going to have
`function my_func() {`
or 
```
function my_func() 
{
```
we have to use the same styling

File: login.php
Comment (metodiew) [inline login.php:38]: what's the reason of having all user fields in a session? Wouldn't be the ID enough and then you can get the details?
What if the user has 20 fields, are we going to use all of them?

If we need these in the session, we might have an array, like this https://stackoverflow.com/questions/21652702/multidimensional-array-in-php-session

**Review by metodiew — CHANGES_REQUESTED** [review]
left a few inline comments, feel free to update them

Also, there is a merge conflict in `index.php` file, so even if the PR was ready, we cannot merge it.

File: index.php
Comment (ivailo-devrix) [reply index.php]: Everywhere in my code I use this style:
function my_func() 
{

File: login.php
Comment (ivailo-devrix) [reply login.php:38]: There is not much data for this project. That's why I decided to keep them. 
My idea is not to re-query the database for query optimization.

I can redo the array as follows:  $_SESSION['user_data']['user_id'] 

When logging in, I can take only the ID and when I need the other data, I can send a new request to the database.

Which of the two options should I implement?

File: includes/theme-compat/header.php
Comment (ivailo-devrix) [reply includes/theme-compat/header.php]: I forgot to format the code.
Is this the correct way it should look:

```
                    <?php if (isset($_SESSION['id_user'])) { ?>
                        <?php if (isset($_SESSION['is_admin']) && $_SESSION['is_admin']) { ?>
                            <li class="menu-item <?php if (isset($page_name) && $page_name == 'dashboard') { ?> current-menu-item <?php } ?> ">
                                <a href="/dashboard.php">Dashboard</a>
                            </li>
                        <?php } ?>
                        <li class="menu-item <?php if (isset($page_name) && $page_name == 'profile') { ?> current-menu-item <?php } ?>">
                            <a href="/profile.php">My Profile</a>
                        </li>
                        <li class="menu-item">
                            <a href="/signout.php">Sign Out</a>
                        </li>
                    <?php } ?>
```

**Comment by ivailo-devrix** [comment]
merge conflict fixed

File: includes/theme-compat/header.php
Comment (metodiew) [reply includes/theme-compat/header.php]: @yanachkov-devix let's update it then :)

File: index.php
Comment (metodiew) [reply index.php]: I can find ~20 instances of the other way around in this PR only

I would strongly suggest to start using `function my_func() {` as WordPress would require the following coding styling

File: login.php
Comment (metodiew) [reply login.php:38]: let's say that Session is not quite reliable information and in 99% of the cases, it's better to call the DB. Not to mention that you can get the user data in one DB query and use it on all pages/files.

Let's update that, as it's way better, trust me

File: index.php
Comment (ivailo-devrix) [reply index.php]: I added in IDE, WordPress coding standards. 
After merging all pool requests, I will make a new batch in which I will only reformat the code without changing anything in terms of functionality.
If I reformat all the code here in this branch it will actually change all the lines in all the files and break the change history

File: login.php
Comment (ivailo-devrix) [reply login.php:38]: I removed the variables from the session that are not sure when and if they will be used at all

**Comment by ivailo-devrix** [comment]
@metodiew I have fixed everything from the comments.
New changes have been uploaded.

File: includes/theme-compat/register-profile.php
Comment (metodiew) [inline includes/theme-compat/register-profile.php]: inline styling alert, we should avoid inline styling. The same comment would be applicable in a few other places in this file

File: includes/theme-compat/register-profile.php
Comment (metodiew) [inline includes/theme-compat/register-profile.php]: a bit weird formatting

File: includes/theme-compat/register-profile.php
Comment (metodiew) [inline includes/theme-compat/register-profile.php]: this is hardly readable, to be honest

**Review by metodiew — CHANGES_REQUESTED** [review]
left a few inline comments

File: includes/theme-compat/register-profile.php
Comment (ivailo-devrix) [reply includes/theme-compat/register-profile.php]: I made some formatting changes, but I'm not sure it has any effect on better readability.

File: includes/theme-compat/register-profile.php
Comment (ivailo-devrix) [reply includes/theme-compat/register-profile.php]: I made some formatting changes, but I'm not sure it has any effect on better readability.

File: includes/theme-compat/register-profile.php
Comment (ivailo-devrix) [reply includes/theme-compat/register-profile.php]: I moved the inline styling to css file

**Comment by ivailo-devrix** [comment]
I made some formatting changes, but I'm not sure it has any effect on better readability.

**Review by metodiew — APPROVED** [review]
synced on Slack - we'll be merging these and @yanachkov-devix will create a few tasks to update and do some refactoring

**Pull request merged**
